R/db-interface.R

```diff
@@ -148,7 +148,9 @@ copy_dm_to <- function(dest, dm, ...,
         abort_copy_dm_to_table_names_duplicated(problem)
       }
 
-      table_names_out <- unclass(DBI::dbQuoteIdentifier(dest_con, table_names_out[src_names]))
+      if (!inherits(table_names_out,"ident_q")) {
```

Comment:
It already works if we pass in an already quoted object, e.g. created by `DBI::SQL()`.

Reply:
This means we no longer need this change?

Reply:
ok, let's properly implement in #432 and until then do as you suggested